.github/workflows/main.yml

```diff
@@ -269,9 +265,7 @@ jobs:
           set -x
           fgrep 'HTML documentation ... no' config.log
           fgrep 'man pages ... no' config.log
-          fgrep 'ch-image(1) ... no' config.log
           fgrep 'with Docker: yes' config.log
-          fgrep 'with ch-image(1): no' config.log
```

Comment:
Why were these lines removed?

Reply:
These were removed early on in the dream of bundling requests and lark. With no dependencies to install it would cause CI to fail because `ch-image` would always be "yes". I think I can roll them back though.